datasets/huggingface.py

```diff
@@ -24,13 +24,19 @@ class DownloadAndPrepareHuggingFaceDatasetJob(Job):
     https://github.com/huggingface/datasets/issues/4179
     """
 
+    __sis_hash_exclude__ = {"split": None, "token": None}
+
+    from datasets import Split
+
     def __init__(
         self,
         path: Union[str, DelayedBase],
         name: Optional[str] = None,
         *,
         data_files: Optional[Any] = None,
         revision: Optional[str] = None,
+        split: Optional[Union[str, Split]] = None,
```

Comment:
Is that really what you would use in a Sisyphus config, this custom `Split` object? Can't this be just a string?

Reply:
i am not sure if we would really use the `Split` object. I guess in most cases we will use a string. But i just added that so it is consistent to the source code documentation.

Reply:
Ok the import is removed now.